src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const titles = document.createElement('h2');
+
+  titles.className = 'title';
+  titles.textContent = title;
+  div.appendChild(titles);
+
+  const descriptions = document.createElement('p');
+
+  descriptions.textContent = description;
+  div.appendChild(descriptions);
+
+  document.body.appendChild(div);
+
+  setTimeout(() => {
+    div.style.visibility = 'hidden';
```

Comment:
Setting the visibility to 'hidden' will only make the element invisible, but it will still take up space on the page. If you want to completely remove the element, you should use 'display: none' or remove the element from the DOM.